File: base/src/main/vsdk/toolkit/render/shaders/PhongBumpShader.java
package vsdk.toolkit.render.shaders;

// GLSL analogue: phongTextureBumpPixelShader.glsl with texture disabled
public final class PhongBumpShader extends LightingShader {
    public PhongBumpShader()
    {
        super(true, false, true);
    }
}

File: typescript/base/src/main/vsdk/toolkit/render/shaders/PhongBumpShader.ts
import { LightingShader } from "./LightingShader.js";

// GLSL analogue: phongTextureBumpPixelShader.glsl with texture disabled
export class PhongBumpShader extends LightingShader {
    public constructor() {
        super(true, false, true);
    }
}
